docs/src/Button.doc.js

```diff
@@ -99,7 +99,7 @@ card(
   <Example
     name="Example"
     defaultCode={`
-<Button text="Medium Sized Button" />
+<Button text="Medium Sized Button" inline />
```

Comment:
Changed most of the examples of the buttons to `inline` since we want to get rid of `fullWidth` buttons in gestalt